packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/AbstractIntegrationTests.java

```diff
@@ -30,7 +30,7 @@
  *
  * @author sbegaudeau
  */
-@SpringJUnitConfig(classes = { IntegrationTestConfiguration.class, SiriusWebStarterConfiguration.class, JDBCConfiguration.class })
+@SpringJUnitConfig(classes = { IntegrationTestConfiguration.class, SiriusWebStarterConfiguration.class, JDBCConfiguration.class, EObjectSerializerTestConfig.class })
```

Comment:
I prefer not to start letting people impact every single tests for just one or two tests. Contrary to its name, it's not even a spring boot configuration, I'm not sur that it's required to work.